### ISSUE TITLE
code actions should have sent / received sounds

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#233717]
When testing the vision extension's generate alt text feature, @jooyoungseo indicated that the progress indication is not enough. We should have audio cues to indicate that the code action was triggered and that the code action was applied.

**Comment by meganrogge** [comment]
For now, we'll re-use the chat response sent / response received cues and get feedback. 

We'll also want feedback on whether there should be a setting to specify which code actions should or shouldn't use these.

**Comment by meganrogge** [comment]
Pls verify with these instructions: https://github.com/microsoft/vscode/issues/233765#issuecomment-2512342569
